platform/nephos/sonic-platform-modules-cig/cs6436-56p/modules/Makefile

```diff
@@ -0,0 +1,5 @@
+obj-m := x86-64-cig-cs6436-56p-cpld.o \
+	x86-64-cig-cs6436-56p-fan.o \
+	x86-64-cig-cs6436-56p-led.o \
+	x86-64-cig-cs6436-56p-psu.o \
+	x86-64-cig-cs6436-56p-sfp.o 
```

Comment:
on other platform, they use optoe driver to support sfp. what is the reason to develop your own sfp driver?

Reply:
As the sfp is controlled with CPLD in the hardware design, it can't be accessed by I2C bus directly. So we can't use the common SFP driver in the Linux system.